src/visualizations/util/getFilterText.js

```diff
@@ -25,6 +30,20 @@ export default function (filters, metaData) {
             )
         ) {
             titleFragments.push(getOuLevelAndGroupText(filter, metaData))
+        } else if (
+            dimensionIs(filter, DIMENSION_ID_PERIOD) &&
+            extraOptions.dashboard
+        ) {
+            titleFragments.push(
+                dimensionGetItemIds(filter)
+                    .map(
+                        (id) =>
+                            getRelativePeriodsName()[id] ||
```

Comment:
Perhaps assign the returned value of this function outside the loop and lookup by id here?

Reply:
There can only be a single `pe` dimension, it will only execute once, so decided to skip the outside const as it's only relevant in this if-block.